### PR TITLE
[SCB-1451] Printing the service information on the log

### PR DESCRIPTION
**Pull request by ZihangJiao**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install -Pit` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: core/src/main/java/org/apache/servicecomb/core/SCBEngine.java [hunk 1]
```diff
@@ -223,10 +228,22 @@ public synchronized void init() {
         }
         status = SCBStatus.FAILED;
         throw new IllegalStateException("ServiceComb init failed.", e);
+      } finally {
+        ServiceInfoPrinter();
       }
     }
   }
 
+  private void ServiceInfoPrinter() {
```
Comment (yhs0092) [inline core/src/main/java/org/apache/servicecomb/core/SCBEngine.java]: - method name should follow CamelCase
- please use verb as method name, e.g. `printServiceInfo` is better than `ServiceInfoPrinter`

File: core/src/main/java/org/apache/servicecomb/core/bootup/AddressInformationCollector.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    private String getCenterInfo(SystemBootstrapInfo systemBootstrapInfo) {
+        if (systemBootstrapInfo == null) {
+            return ("not exist");
+        } else {
+            return systemBootstrapInfo.getAccessURL().toString();
+        }
+    }
+
+}
```
Comment (yhs0092) [inline core/src/main/java/org/apache/servicecomb/core/bootup/AddressInformationCollector.java]: The indent of some files a different from our code style, please reformat it : )
And you can find code style config files [here](https://github.com/apache/servicecomb-java-chassis/tree/master/etc)

File: core/src/main/java/org/apache/servicecomb/core/SCBEngine.java (hunk 1 above)
Comment (ZihangJiao) [reply core/src/main/java/org/apache/servicecomb/core/SCBEngine.java]: The name of the method have been changed to printServiceInfo.

File: core/src/main/java/org/apache/servicecomb/core/bootup/AddressInformationCollector.java (hunk 2 above)
Comment (ZihangJiao) [reply core/src/main/java/org/apache/servicecomb/core/bootup/AddressInformationCollector.java]: The code is formatted

**Pull request merged**